tensorflow_addons/utils/keras_utils.py

```diff
@@ -19,10 +19,50 @@
 
 import tensorflow as tf
 
-# TODO: find public API alternative to these
-from tensorflow.python.keras.utils import conv_utils  # pylint: disable=unused-import
-
 
 def register_keras_custom_object(cls):
     tf.keras.utils.get_custom_objects()[cls.__name__] = cls
     return cls
+
+
+def normalize_tuple(value, n, name):
+    """Transforms a single integer or iterable of integers into an integer
+    tuple. Ported from tensorflow.python.keras.util.
```

Comment:
How about splitting the paragraph to two single lines? Because summary line should be one line, as required in https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings

Reply:
Done. Thanks!